Makefile.am

```diff
@@ -25,7 +25,6 @@ libfastjsoninclude_HEADERS = \
 
 libfastjson_la_LDFLAGS = -version-info 3:0:0 -no-undefined @JSON_BSYMBOLIC_LDFLAGS@ \
 	-export-symbols-regex '^fjson_.*'
-libfastjson_la_CPPFLAGS = -Werror
```

Comment:
Ok, will prep a separate PR for this once this PR has been merged.
Do you prefer if a copy of ax_compiler_flags.m4 is included in m4 or do we rely on autoconf-archive being installed for those who wish to build from git?